src/NuGet.Clients/NuGet.Tools/NuGet.Tools.csproj

```diff
@@ -20,10 +20,8 @@
     <VSSDKTargetPlatformRegRootSuffix>Exp</VSSDKTargetPlatformRegRootSuffix>
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
-    <FileUpgradeFlags>
-    </FileUpgradeFlags>
-    <UpgradeBackupLocation>
-    </UpgradeBackupLocation>
+    <FileUpgradeFlags />
+    <UpgradeBackupLocation />
     <OldToolsVersion>14.0</OldToolsVersion>
```

Comment:
Should we update this to 15.0?

Reply:
We can, but I think this is irrelevant. 
It's only important when migrating the project.